lib/authentication/keycloak/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+COPY ./tmp/* /opt/jboss/keycloak/standalone/
```

Comment:
What are these files supposed to be that it's copying? When I build this, I get `COPY failed: no source files were specified`

Reply:
Due to the way this AAI PR is written, it is not compatible with the current CanDIGv2 build and deployment specs. These require their own `compose-authx-setup` method to be called to initialize, build and deploy the AAI stack. The idea is for now to merge this branch and propose new fixes in a later PR.